apps/condo/domains/ticket/constants/common.js

```diff
@@ -19,6 +19,7 @@ const TICKET_SOURCE_TYPES = {
     SOCIAL_NETWORK: 'social_network',
     MOBILE_APP_STAFF: 'mobile_app_staff',
     MOBILE_APP_RESIDENT: 'mobile_app_resident',
+    IMPORT: 'import',
```

Comment:
May be you need to specify type more precisely. What if some B2B app will have import feature as well? Should we differentiate this cases?

Reply:
change to `CRM_IMPORT: 'crm-import'`